packages/modular-scripts/src/config/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
What's the reason for this being a js file and now TypeScript?

Reply:
This started out as the `jest-config.js` file which I reallocated and renamed. I kept it as a js file because craco does not provide types so we'd have to declare @craco as a module and set createJestConfig as an export within modular-scripts. This is why I added the `.js` to the build script. I personally wasn't sure which route to go. Both are not ideal.